src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderTransactionHistoryDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTransactionHistory;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@ApiModel(description = "Offender transaction details")
+@Data
+@EqualsAndHashCode
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OffenderTransactionHistoryDto {
+
+    @ApiModelProperty(value = "Offender Id", example = "1", position = 1)
+    private Long offenderId;
+
+    @ApiModelProperty(value = "Transaction Id", example = "1", position = 2)
+    private Long transactionId;
+
+    @ApiModelProperty(value = "Transaction Sequence", example = "1", position = 3)
+    private Long transactionEntrySequence;
+
+    @ApiModelProperty(value = "Transaction Date", example = "2020-12-11", position = 4)
+    private LocalDate entryDate;
+
+    @ApiModelProperty(value = "Transaction Type", example = "???", position = 5) //todo
```

Comment:
Do you mean to check this in?

Reply:
will try and find out what Transaction Types there are and remove